types/json/binary.go

```diff
@@ -169,6 +169,16 @@ func (bj BinaryJSON) GetString() []byte {
 	return bj.Value[lenLen : lenLen+int(strLen)]
 }
 
+// GetKeys gets the keys of the object
+func (bj BinaryJSON) GetKeys() BinaryJSON {
+	count := bj.getElemCount()
+	ret := make([]BinaryJSON, 0, count-1)
```

Comment:
should it be `count` rather than `count-1`?

Reply:
sure🌚